### ISSUE TITLE
MONAI FL integration

### ISSUE THREAD

**Issue by sharkovsky** [fedbiomed/fedbiomed#908]
Following our discussion with the MONAI FL WG, we would like to submit a pull request with a [tutorial](https://github.com/Project-MONAI/tutorials/tree/main/federated_learning) showcasing the integration of Fed-BioMed with their [MONAI FL API](https://docs.monai.io/en/stable/fl.html). 

Ping me for more information and to express interest!

**Comment by sharkovsky** [comment]
@luciainnocenti are you interested?

**Comment by luciainnocenti** [comment]
> @luciainnocenti are you interested?

Yes, absolutely. We can have a meeting next week and start planning it
